extensions/wikia/Chat2/ChatBanListSpecial.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @package MediaWiki
+ * @subpackage SpecialPage
+ * @author Piotr Molski <moli@wikia-inc.com> for Wikia.com
+ * @version: 1.0
+ */
+class ChatBanListSpecial extends SpecialRedirectToSpecial
```

Comment:
Maybe it would be a good idea to subclass `WikiaSpecialPageController` instead. This way you'll be able to deal with templating much more easily instead of having to use the old and deprecated `EasyTemplate` system.